libraries/AP_ICEngine/AP_ICEngine.h

```diff
@@ -60,6 +60,11 @@ class AP_ICEngine {
 
     enum ICE_State state;
 
+    // filter for RPM value
+    float rpm_value;
+    LowPassFilterFloat _rpm_filter;
+    float filtered_rpm_value;
```

Comment:
complete feature creep, but I wonder if we should just have a filter with param value for each RPM sensor...

Reply:
although that would be bad if you wanted to set the notch filter from the RPM and use this new governor, you would need completely different filter frequencies.

Reply:
I didn't want to add the extra size for each rpm source now that we can have more than 2. If we start using a LPF RPM value more, then it would make more sense, correct?